optimizely_server_side.gemspec

```diff
@@ -2,7 +2,7 @@ $:.push File.expand_path("../lib", __FILE__)
 
 Gem::Specification.new do |s|
   s.name        = 'optimizely_server_side'
-  s.version     = '1.2.2'
+  s.version     = '1.2.3'
   s.date        = '2018-05-22'
```

Comment:
Since you are here can you remove this line `s.date = '2018-05-22'`. We actually don't need it as it [defaults to UTC](https://ruby-doc.org/stdlib-2.2.3/libdoc/rubygems/rdoc/Gem/Specification.html#method-i-date)

Reply:
will do